system_test/aest_sync_SUITE.erl

```diff
@@ -705,14 +707,48 @@ net_split_mining_power(Cfg) ->
 
     ok.
 
-quick_start_stop(Cfg) ->
-    setup_nodes(cluster([n1, n2], #{}), Cfg),
+abrupt_stop_new_node(_Cfg) ->
+    {skip, database_restart_needs_fix};
+abrupt_stop_new_node(Cfg) ->
+    RepairTimeout = 30000, % Time allowed for node to repair DB and finish sync
+    Nodes = [n1, n2],
+    setup_nodes(cluster(Nodes, #{}), Cfg),
+    % Start both nodes
+    [start_node(N, Cfg) || N <- Nodes],
+    % Stop node 2 abruptly
+    stop_node(n2, 2000, Cfg),
```

Comment:
Using stop will effectively kill the container when the timeout expires. The alternative equivalent to this would be:
```
timer:sleep(2000),
stop_node(n2, 0, Cfg),
```
If we kill it too early, the node will not have time to start.

Reply:
> Using stop will effectively kill the container when the timeout expires.

On a fast machine the timeout may not expire - i.e. clean stop may complete before 2s. Hence this test would stop gently, not abruptly (test case name is `abrupt_stop_new_node`).

> The alternative equivalent to this would be:
> 
> timer:sleep(2000),
> stop_node(n2, 0, Cfg),

It is quite difficult to define equivalent with timeout outside the call. Also the [stop retries 5 times with the same timeout](https://github.com/aeternity/epoch/blob/acc7fd0651a3e6ac79e5195126494519f25fbe6a/system_test/helpers/aest_docker.erl#L460).

> If we kill it too early, the node will not have time to start.

Yes, test shall check node started before stopping it.


It feels like test here shall do:
```
wait_for_startup(n2, 0, Cfg),
aest_nodes:kill_node(n2, Cfg)
```

Reply:
That would be the exact opposite of an abrupt start. We currently trigger the DB corruption problem on most machines with this setup. I suggest we keep it until we come up with something better.